lib/mongodb.js

```diff
@@ -39,6 +40,7 @@ function ObjectID(id) {
   }
 }
 
+exports.generateMongoDBURL = generateMongoDBURL;
```

Comment:
No there could be potential problems if we take out code from the `initialize` function and put it on `MongoDB` constructor just to avoid having two other export functions. Could be a breaking change for the `initialize` function. It looks good to me how it is right now.